geopandas/plotting.py

```diff
@@ -325,55 +315,48 @@ def plot_dataframe(df, column=None, cmap=None, color=None, ax=None,
 
     Parameters
     ----------
-
     df : GeoDataFrame
         The GeoDataFrame to be plotted.  Currently Polygon,
         MultiPolygon, LineString, MultiLineString and Point
         geometries can be plotted.
-
     column : str (default None)
         The name of the column to be plotted. Ignored if `color` is also set.
-
     cmap : str (default None)
         The name of a colormap recognized by matplotlib.
-
+    color : str (default None)
+        If specified, all objects will be colored uniformly.
+    ax : matplotlib.pyplot.Artist (default None)
```

Comment:
I was just re-organizing the parameter descriptions to match the order they appear. In doing so, I went ahead and made whitespace consistent with the standards elsewhere.